week-1/L-functions-nested/exercise2.js

```diff
@@ -3,3 +3,17 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+function Capital_Letters() {
+  mentor1.toUpperCase();
+  mentor2.toUpperCase();
+  return mentor1;
+}
+
+function Display_Name() {
+  return Capital_Letters();
+}
+
+var message = Display_Name();
```

Comment:
Here, we want to be able to write `var message = displayName(mentor1);` instead of `var message = Display_Name();`. This way you can use the `displayName` function with many different variables like `mentor3`, `mentor4` and `mentor5`